### PR TITLE
Initial changes

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[Min/max validation using a Calculated Summary (3 SP)](https://trello.com/c/JHx4JvaP/5270-min-max-validation-using-a-calculated-summary-3-sp)

File: docker-compose-schema-validator.yml [hunk 1]
```diff
@@ -3,14 +3,14 @@ version: '3'
...
-    image: onsdigital/eq-questionnaire-validator:${TAG}-ajv
+    image: onsdigital/eq-questionnaire-validator:calculated-summary-min-max-ajv
     networks:
       - eq-schema
     ports:
       - 5002:5002
 
   py-validator:
-    image: onsdigital/eq-questionnaire-validator:${TAG}
+    image: onsdigital/eq-questionnaire-validator:calculated-summary-min-max
```
Comment (MebinAbraham) [inline docker-compose-schema-validator.yml:13]: Reminder to revert before merge.

File: schemas/test/en/test_calculated_summary.json [hunk 2]
```diff
@@ -460,6 +460,69 @@
...
+                            "type": "Question",
+                            "id": "set-min-max-block",
+                            "skip_conditions": [
+                                {
+                                    "when": [
+                                        {
+                                            "id": "skip-fourth-block-answer",
+                                            "condition": "equals",
+                                            "value": "No"
+                                        }
```
Comment (MebinAbraham) [inline schemas/test/en/test_calculated_summary.json:474]: We should use new rules structure.

File: tests/functional/spec/features/calculated_summary.spec.js [hunk 3]
```diff
@@ -239,7 +251,14 @@ describe("Feature: Calculated Summary", () => {
       );
 
       $(CurrencyTotalPlaybackPageSkippedFourth.submit()).click();
-
+      $(UnitTotalPlaybackPage.submit()).click();
+      $(PercentageTotalPlaybackPage.submit()).click();
+      $(NumberTotalPlaybackPage.submit()).click();
+      $(CalculatedSummaryTotalConfirmation.submit()).click();
+      expect(browser.getUrl()).to.contain(SetMinMaxBlockPage.pageName);
+      $(SetMinMaxBlockPage.setMinimum()).setValue(100.0);
```
Comment (MebinAbraham) [inline tests/functional/spec/features/calculated_summary.spec.js:259]: Same as the number spec, we should try to submit and ensure we get an updater error message. Will need to be tested for minimum and maximum.

**Review by MebinAbraham — COMMENTED** [review]
Looks good, minor comments.

**Review by petechd — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]
